public/app/core/components/sidemenu/BottomNavLinks.tsx

```diff
@@ -67,7 +67,7 @@ class BottomNavLinks extends PureComponent<Props, State> {
             </li>
           )}
 
-          <OrgSwitcher onDismiss={this.toggleSwitcherModal} isOpen={showSwitcherModal} />
+          {showSwitcherModal && <OrgSwitcher onDismiss={this.toggleSwitcherModal} />}
```

Comment:
Thanks for this @torkelo ! I consider this a more elegant way of toggling visibility when no animation is required.